src/prefect/cli/cloud.py

```diff
@@ -294,6 +294,9 @@ async def login(
         "Creating a profile for this Prefect Cloud login. Please specify a profile name: "
     )
 
+    if cloud_profile_name == "":
```

Comment:
Great catch!

This is definitely an edge case, but we might want to strip the user's input of leading/trailing spaces:

```suggestion
    cloud_profile_name = cloud_profile_name.strip()
    if cloud_profile_name == "":
```

Reply:
great idea!